crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1056,6 +1056,33 @@ impl SubCommand for CliModel {
                         .value_parser(["data", "graphql"])
                         .default_values(&["data", "graphql"])
                         .help("which API endpoints to offer")
+                    )
+                    .arg(
```

Comment:
Should these OPA options be on the `serve-api` subcommand only? Others like `import` may also need them.